.gitignore

```diff
@@ -1 +1,2 @@
+.idea/
```

Comment:
please revert this; your personal IDE choices should go in your global gitconfig, instead of polluting individual repos